examples/pytorch/seal_ogbl/README.md

```diff
@@ -0,0 +1,43 @@
+# SEAL Implementation for OGBL in DGL
+
+Introduction
+------------
+This is an example of implementing [SEAL](https://arxiv.org/pdf/2010.16103.pdf) for link prediction in DGL. Some parts are migrated from [https://github.com/facebookresearch/SEAL_OGB](https://github.com/facebookresearch/SEAL_OGB).
+
+Requirements
+------------
+[PyTorch](https://pytorch.org/), [DGL](https://www.dgl.ai/), [OGB](https://ogb.stanford.edu/docs/home/), and other python libraries: numpy, scipy, tqdm, sklearn, etc.
+
+Usages
+------
+Run the scripts in `./scripts` folder, e.g.
+```bash
+bash scripts/run_ppa.sh
+```
+
+Results
+-------
+
+|              | ogbl-ppa (Hits@100) | ogbl-collab (Hits@50) | ogbl-ddi (Hits@20) | ogbl-citation2 (MRRd) |
+|--------------|---------------------|-----------------------|--------------------|---------------------|
+| Paper Test results |  48.80%&plusmn;3.16% |    64.74%&plusmn;0.43% | 30.56%&plusmn;3.86% |   87.67%&plusmn;0.32% |
+| Our Test results |  49.48%&plusmn;2.52% |    64.23%&plusmn;0.57% | 27.93%&plusmn;4.19% |   86.29%&plusmn;0.47% |
```

Comment:
Perhaps change "Paper Test results" -> "Paper Test Results" or "Paper test results" for consistency. Same for "Our Test results"